### DISCUSSION TITLE
Definición proyecto de eliminación adjuntos temporales (papelera) Correspondencia y Archivo

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#37] [General]
Opción que permitirá a los clientes eliminar los archivos que han cumplido su tiempo de retención y adjuntos de correspondencias de años anteriores que ya no requieran ser conservados al perder sus valores documentales.

**Comment by aleja550** [comment]
Esto nunca se realizo.
